server/url_params.go

```diff
@@ -63,6 +67,77 @@ func ExtractParametersFromUrl(url *url.URL) (params URLParameters, err error) {
 		params.pref.Builders = targetBuildersQuery
 	}
 
+	refundAddressQuery, ok := url.Query()["refund"]
+	if ok {
+		if len(refundAddressQuery) == 0 {
+			return params, ErrIncorrectRefundQuery
+		}
+
+		var (
+			addresses = make([]common.Address, len(refundAddressQuery))
+			percents  = make([]int, len(refundAddressQuery))
+		)
+
+		for i, refundAddress := range refundAddressQuery {
```

Comment:
**suggestion:**  Can you add small, concise comment that provides **example for structure of refund address query**? 

A link to any existing spec would also be great. Right now it's not clear what we're expecting the valid input to look like.